File: Alpata.Case.Domain/Models/Meeting.cs
using Alpata.Case.Domain.Base;

namespace Alpata.Case.Domain.Models
{
    public class Meeting : Entity
    {
        public string Name { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public string? Explanation { get; set; }
        public virtual ICollection<MeetingAttachment> Attachments { get; set; }
    }
}

File: Alpata.Case.Web/ClientApp/src/app/meeting/model/meeting.ts
import { MeetingAttachment } from "./meetingAttachment";

export class Meeting {
    constructor(
        id: string
    ) {
        this.id = id
    }
    id: string;
    name!: string;
    startTime!:string|Date;
    endTime!:string|Date;
    explanation!:string;
    attachments:MeetingAttachment[] = [];
}